man/docgen

```diff
@@ -531,6 +576,34 @@ def completion_output(targets, substs, template_file):
 
     print_template(template_file, substs, content)
 
+def carray_output(targets, substs, template):
+
+    content = "\nstatic const char *params[] = {\n"
+
+    for t in targets:
+        content += t.carray_output(False)
+
+    content += "};\n\nstatic const char *params_with_args[] = {\n"
+
+    for t in targets:
+        content += t.carray_output(True)
+
+    content += "};\n\n#define HELP_STRING \""
+
+    c = dict(categories)
+
+    for t in targets:
+        # no video and obscure category
+        if t != c["video"] and t != c["obscure"]:
```

Comment:
Why no video?

Reply:
I don't think we have a video option important enough to include in the `-help` output. However, I'm not sure.

Reply:
Okay, the most important options already have their switches in the menu.